app/services/decorators/v5/property_decorator.rb

```diff
@@ -19,7 +19,7 @@ def payload
           main_home: @record.main_home,
           shared_with_housing_assoc: @record.shared_with_housing_assoc,
           transaction_allowance: @record.transaction_allowance,
-          allowable_outstanding_mortgage: @record.allowable_outstanding_mortgage,
+          allowable_outstanding_mortgage: @record.outstanding_mortgage,
```

Comment:
If we're making this change here, and formalising that `allowable_outstanding_mortgage` isn't used and doesn't matter any more, shall we clear up PropertyCalculator by stopping bothering with keeping a running total of `property_maximum_mortgage_allowance`?